extensions/wikia/AssetsManager/config.php

```diff
@@ -922,7 +924,8 @@
 		'#group_imglzy_js',
 		'#group_spotlights_js',
 		'//resources/wikia/libraries/ghostwriter/gw.min.js',
-		'//extensions/wikia/GlobalNotification/GlobalNotification.js',
+		'#group_banner_notifications_js',
+		'//skins/shared/scripts/onScroll.js',
```

Comment:
After offline discussion with @kvas-damian , we decided to keep things the way there are and consider resolving dependency management for AssetManager as a seperate cause.